lib/services/ngsi/ngsiService.js

```diff
@@ -36,6 +36,7 @@ var request = require('request'),
     utils = require('../northBound/restUtils'),
```

Comment:
CNR entry should be added

Reply:
Fixed with 0f341d164437b4f2b9876ba8acf56d25afa5cefa